Userland/Libraries/LibCore/EventLoop.cpp

```diff
@@ -6,6 +6,7 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include "AK/RefPtr.h"
```

Comment:
clang powaaaaaaa

Reply:
jesus christ what is this program thinking